lib/instructions/instruction_metrics.c

```diff
@@ -277,7 +277,7 @@ static struct evmc_instruction_metrics istanbul_metrics[256] = {
     /*                = 0xfc */ {UNDEFINED, 0, 0},
     /*         REVERT = 0xfd */ {ZERO, 2, 0},
     /*        INVALID = 0xfe */ {ZERO, 0, 0},
-    /*        SUICIDE = 0xff */ {5000, 1, 0},
+    /*   SELFDESTRUCT = 0xff */ {5000, 1, 0},
```

Comment:
Wow this wasn't done yet? :)

Reply:
Only in comments. This was created by transforming Aleth's table / enum.